lib/tasks/data_import/services/README.md

```diff
@@ -28,62 +28,62 @@ In order to add multiple locations to a service you would create one entry for y
+| `suitabilities`                                                                                                                 | semi colon (;) delimited list   |  List of suitabilities for this location |  |  
+| `is_local_offer`                                                                                                                | Boolean  | Is this service part of the local offer   |  |  
+| `send_needs_support`                                                                                                            | semi colon (;) delimited list  | Which SEND needs can you support?  |  |  
+| `support_description`                                                                                                           | Text | Description of support provided  |  |  
+| `recent_send_report`                                                                                                            | Text (url)  | Link to most recent SEND report  |  |  
+| `outcomes`                                                                                                                      | Text | What outcomes does your setting, service or activity aim to achieve for children and young people with SEND and their families?  |  |  
+| `recent_send_training`                                                                                                          | Text | What recent SEND-specific training has been completed by your staff and/or volunteers?  |  |  
+| `parental_involvement`                                                                                                          | Text  |  How do you involve parents and how can I get involved? |  |  
+| `information_sharing`                                                                                                           | Text  |  How will you share information with me about my child’s progress? What is additional for children with SEND? |  |  
+| `environment_accessibility`                                                                                                     | Text | How accessible is the environment (indoors and outdoors)?  |   |  
+| `how_to_start`                                                                                                                  | Text  |  How can children and young people with SEND and their families start to use your service or activity? |  |  
+| `future_plans`                                                                                                                  | Text  | What future plans do you have for developing your SEND provision?  |  |  
+| `custom_<custom_field_type>_field_name`                                                                                         | depends  | Format for custom fields  |  |  
```

Comment:
I don't understand how this is meant to be used - how do we know which custom field we're adding service meta for - for example if there is more than one `text` type custom field?

Reply:
those fields are for the SEND needs section.

Custom fields is this bit
`custom_<custom_field_type>_field_name | depends | Format for custom fields
-- | -- | --
`



if we had a custom field called "Type of guinea pig" that was free text (created previously with the custom import script or already in the site) then the column name would be `custom_text_type_of_guinea_pig` if we had one that was a select field for "What is the best vegetable for piggies?" there would be a column `column_select_what_is_the_best_vegetable_for_piggies` with content like "cucumbers;red peppers"

Reply:
This makes sense to me now! I've added an example to the README.